crates/wasmcloud-host/src/host.rs

```diff
@@ -222,8 +222,10 @@ impl Host {
     /// to provide some form of parking or waiting (e.g. wait for a Ctrl-C signal).
     pub async fn start(&self) -> Result<()> {
         let kp = KeyPair::new_server();
+        eprintln!("Host starting ...");
```

Comment:
Let's change these `eprintln!` calls to log macro invocations, I think that these can be at the `debug` level as they would be useful for debugging, but potentially too noisy for `info`.

Reply:
We discussed this one on the zoom call, these were debug statements that he's going to remove.

Reply:
fixed in this branch